app/controllers/api/stories_controller.rb

```diff
@@ -68,7 +68,7 @@ def included(relation)
   end
 
   def scoped(relation)
-    relation.published.visible
+    relation.published.network_visible.series_visible
```

Comment:
Can you explain this?  I'm not sure I understand what both those statements are doing, assuming the user is supplying only one of `network_id` or `series_id`.